components/admin_console/admin_definition.jsx

```diff
@@ -512,11 +515,11 @@ export default {
                                 </b>
                             ),
                             recycleDatabaseConnections: (
-                                <a href='../advanced/database'>
+                                <a href='../environment/database'>
                                     <b>
                                         <FormattedMessage
                                             id='admin.reload.reloadDescription.recycleDatabaseConnections'
-                                            defaultMessage='Database > Recycle Database Connections'
+                                            defaultMessage='Environment > Database > Recycle Database Connections'
```

Comment:
Should update corresponding translation text at i18n/en.json.

Same case for `admin.support.enableTermsOfServiceHelp`, `admin.ldap.groupFilterFilterDesc` and other help texts that were changed.

Reply:
I'll re-check those, thanks @saturninoabril